qchem/pennylane_qchem/qchem.py

```diff
@@ -504,7 +504,7 @@ def generate_hamiltonian(
     **Example usage:**
 
     >>> generate_hamiltonian('h2', 'h2.xyz', 0, 1, 'sto-3g')
-    (<pennylane.beta.vqe.vqe.Hamiltonian object at 0x7fa406966e48>, 4)
+    (<pennylane.vqe.vqe.Hamiltonian object at 0x7fa406966e48>, 4)
```

Comment:
why does `vqe` show up twice??

Also, won't it show up at top-level now? i.e., `<pennylane.Hamiltonian object...>`

Reply:
I just double checked, it's because the full import path is `pennylane/vqe/vqe.py::Hamiltonian`. We can 'mock' this by definining the `__repr__` method on the `Hamiltonian` object, or alternatively modify this docstring example.

My preference is the latter, because by default `__repr__` always show the full import path (so it affects every class in Python where the true location does not match the import path). e.g.
```python
>>> qml.PauliX(0)
pennylane.ops.qubit.PauliX at 0x7fb53fe36d68>
```

Reply:
Change the docstring example to:
```python

    **Example usage:**

    >>> H, n_qubits = generate_hamiltonian('h2', 'h2.xyz', 0, 1, 'sto-3g')
    >>> print(n_qubits)
    4
    >>> print(H)
    (-0.04207897647782188) [I0]
    + (0.17771287465139934) [Z0]
    + (0.1777128746513993) [Z1]
    + (-0.24274280513140484) [Z2]
    + (-0.24274280513140484) [Z3]
    + (0.17059738328801055) [Z0 Z1]
    + (0.04475014401535161) [Y0 X1 X2 Y3]
    + (-0.04475014401535161) [Y0 Y1 X2 X3]
    + (-0.04475014401535161) [X0 X1 Y2 Y3]
    + (0.04475014401535161) [X0 Y1 Y2 X3]
    + (0.12293305056183801) [Z0 Z2]
    + (0.1676831945771896) [Z0 Z3]
    + (0.1676831945771896) [Z1 Z2]
    + (0.12293305056183801) [Z1 Z3]
    + (0.176276408043196) [Z2 Z3]
```